projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -108,6 +108,47 @@ <h3 *ngIf="content.title" class="cx-card-title">
             >{{ action.name }}</a
           >
         </div>
+        <div *cxFeatureLevel="'4.2'" [ngSwitch]="action.event">
+          <button
+            *ngSwitchCase="'delete'"
+            class="cx-card-link card-link btn-link cx-action-link"
+            (click)="delete()"
+            (keydown.enter)="delete($event)"
```

Comment:
We shouldn't need to use `(keydown.enter)` now that these are buttons. `(click)` should handle any enter keypresses.